tests/test_token_gen.py

```diff
@@ -471,6 +484,24 @@ def test_revoked_token_check_revoked(self, user_mgt_app, revoked_tokens, id_toke
             auth.verify_id_token(id_token, app=user_mgt_app, check_revoked=True)
         assert str(excinfo.value) == 'The Firebase ID token has been revoked.'
 
+    @pytest.mark.parametrize('id_token', valid_tokens.values(), ids=list(valid_tokens))
+    def test_disabled_user_check_revoked(self, user_mgt_app, user_disabled, id_token):
+        _overwrite_cert_request(user_mgt_app, MOCK_REQUEST)
+        _instrument_user_manager(user_mgt_app, 200, user_disabled)
+        with pytest.raises(auth.UserDisabledError) as excinfo:
+            auth.verify_id_token(id_token, app=user_mgt_app, check_revoked=True)
+        assert str(excinfo.value) == 'The user record is disabled.'
+
+    @pytest.mark.parametrize('id_token', valid_tokens.values(), ids=list(valid_tokens))
+    def test_disabled_revoked_check_revoked(
```

Comment:
Nit: We can call this `test_check_disabled_before_revoked()` and drop the comment if you like.

Reply:
Done